autotest/interface/pipeline/test_pipeline_longtext_func.py

```diff
@@ -79,41 +80,50 @@ def stream_infer_basic(config, model, log_name):
 
 @pytest.mark.gpu_num_1
 @pytest.mark.parametrize(
-    'model', ['internlm/internlm2-chat-7b', 'internlm/internlm2-chat-1_8b'])
-def test_long_test_passkey_tp1(config, model, worker_id):
+    'model', ['internlm/internlm2-chat-7b', 'Qwen/Qwen2-7B-Instruct'])
+@pytest.mark.parametrize('backend', ['turbomind'])
+def test_long_test_passkey_tp1(config, model, backend, worker_id):
     log_name = ''.join(['pipeline_longtext_passkey_', worker_id, '.log'])
     if 'gw' in worker_id:
         os.environ['CUDA_VISIBLE_DEVICES'] = get_cuda_id_by_workerid(worker_id)
-    p = Process(target=passkey_retrival, args=(config, model, log_name))
+    p = Process(target=passkey_retrival,
```

Comment:
It is a long-context test on internlm2-chat-7b and qwen2-7b-instruct, right?
But, turbomind engine doesn't support slide window for qwen2-7b-instruct